src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -144,11 +145,17 @@ private void clickByJS(By selector) {
     new JavascriptActions(driver).execute("arguments[0].click();", driver.findElement(selector));
   }
 
+  private void clickOnBottomLeftCornerOfElement(WebElement element) {
+    int middlePointOfWebElement = element.getSize().height / 10;
+    int leftSideOfWebElement = element.getSize().width / 3;
+    Actions builder = new Actions(driver);
+    builder.moveToElement(element, middlePointOfWebElement,leftSideOfWebElement).click().build().perform();
+
+  }
+
   private void clickElementInsideAd(By selector) {
     usingAdFrame(() -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)